src/modules/items-count.js

```diff
@@ -0,0 +1,17 @@
+// const count = () => {
+//   const box = document.getElementById('list-home');
+
+//   const directChildren = box.children.length;
+//   const cat = document.getElementById('meals');
+//   cat.innerHTML = `Meals(${directChildren + 1})`;
+// };
+
+// export default count;
+
+const mealCounter = document.getElementById('meals');
+
+export default function countMeals(meals) {
+  if (mealCounter !== null) {
+    mealCounter.innerHTML = `<p>Meals(${meals.length})</p>`;
+  }
+}
```

Comment:
- [ ] As suggested by previous reviewer please use arrow functions instead of function keyword as its been depreciated.